hadoop-ozone/recon-codegen/src/main/java/org/hadoop/ozone/recon/schema/ContainerSchemaDefinition.java

```diff
@@ -49,7 +49,8 @@
     MISSING,
     UNDER_REPLICATED,
     OVER_REPLICATED,
-    MIS_REPLICATED
+    MIS_REPLICATED,
```

Comment:
Why do we need this? It is still ambiguous and I see the value is not set right now. Unhealthy is already a catch-all.

Reply:
I wanted to introduce a new query state that has Recon return containers that have all replicas as UNH, as opposed to the more generic 'UNDER_REPLICATED' state that typically denotes a transient state. I have modified the enum field to reflect it better. This can be queried as follows

/containers/unhealthy/UNDER_REPLICATED
/containers/unhealthy/ALL_REPLICAS_UNHEALTHY
/containers/unhealthy --> will return containers in any bad state.

Reply:
Actually, my concerns were about MIS_REPLICATED state, sorry for not being specific with the earlier comment, since we already have unhealthy, as a user mis_replicated seems a bit incomprehensible.

Reply:
Oh, that is not part of this patch. MIS_REPLICATED is the name SCM (& Recon) uses to label containers that have sufficient replicas, but do not conform to the topology requirements.

Reply:
Got it, can we make sure this state is documented in the code, will be confusing for users.